backend/tests/integration/disambiguation/test_tasks.py

```diff
@@ -84,6 +85,43 @@ def test_disambiguate_signatures_cluster_with_0_authors(
     assert expected_ref == record["authors"][0]["record"]["$ref"]
 
 
+@freeze_time("2019-02-15")
+def test_disambiguate_signatures_cluster_creates_author_with_facet_author_name(
```

Comment:
how is this different from the async test you also added?

Reply:
In the async one I'm checking the facet_author_name on the literature record once an author is linked by disambiguation (I moved it to async so I don't trigger the indexing myself, can also be moved to the integration tests and I would trigger the indexing, wdyt?), I guess the naming is misleading. In the integration one, I'm checking the facet_author_name on a newly created author (from the author's detail page)